Package.swift

```diff
@@ -38,8 +38,8 @@ let package = Package(
     // Dependencies declare other packages that this package depends on.
     // .package(url: /* package url */, from: "1.0.0"),
     .package(url: "https://github.com/kateinoigakukun/JavaScriptKit.git", .revision("c90e82f")),
-    .package(url: "https://github.com/MaxDesiatov/Runtime.git", .branch("wasi-build")),
-    .package(url: "https://github.com/MaxDesiatov/OpenCombine.git", .branch("observable-object")),
+    .package(url: "https://github.com/MaxDesiatov/Runtime.git", .branch("wasi-build-5.3")),
+    .package(url: "https://github.com/MaxDesiatov/OpenCombine.git", .branch("observable-object-5.3")),
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Package.swift;
  Line: 42;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 102 characters (`line_length`)